### DISCUSSION TITLE
Is it possible to make all keywords **bold** and _italic_

### DISCUSSION THREAD

**Discussion by aslpavel** [zed-industries/zed#16611] [Feature Requests]
In VSCode it is possible to make all keywords bold and italic by adding
```json
    "keywords": {
      "fontStyle": "bold italic"
    },
```
I was wandering if it is possible to do something similar in zed? It is greatly improving readability for me

**Comment by timmclean** [comment]
Adding this to my `~/.config/zed/settings.json` worked for me:

```
  "experimental.theme_overrides": {
    "syntax": {
      "keyword": {
        "font_style": "italic",
        "font_weight": 700
      }
    }
  }
```

I figured out what to override by looking at a [theme file](https://github.com/zed-industries/zed/blob/56f69be2e772a39a0489bc736b35e08d04b79299/assets/themes/one/one.json#L557).

**Reply by aslpavel** [reply]
Nice, thank you!
